tests/elliptic_curves/test_ec_arithmetic.py

```diff
@@ -31,7 +30,14 @@ class Secp256k1:
     )
     test_script = EllipticCurveFq(q=modulus, curve_a=0)
     test_script_unrolled = EllipticCurveFqUnrolled(q=modulus, ec_over_fq=test_script)
-    test_positions = False
+    # All possible combinations: ± P ± Q are tested. Refer to ./util.py
+    positions_addition = [
+        {"modulus": 25, "gradient": 20, "P": 14, "Q": 5},
+    ]
+    # All possible combinations: ± 2P are tested. Refer to ./util.py
+    positions_doubling = [
+        {"modulus": 25, "gradient": 20, "P": 14},
+    ]
```

Comment:
maybe you can move that directly to the argument of generate_test_data for conciseness

Reply:
Restructured according to slow tests.